src/app/tooltips/tooltips.view.ts

```diff
@@ -54,7 +54,7 @@ export class TooltipsView {
                             // width: 'fit-content',
                         },
                         innerText: tooltipText,
-                        children: [new CloseTooltipView(divId)],
+                        children: [new CloseTooltipView(divId, _tooltipState)],
```

Comment:
why do you prefix '_tooltipState' variable with '_' ?
The regular usage is usually for:
*  unused variable
*  private variable (there is some debate on that, we should try to get consistency on that; typescript team do not prefix private variable with '_', https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines)